crates/sdk/src/lib.rs

```diff
@@ -268,6 +270,23 @@ impl<E: StarkFriEngine<SC>> GenericSdk<E> {
         Ok(proof)
     }
 
+    pub fn generate_e2e_stark_proof<VC: VmConfig<F>>(
+        &self,
+        app_pk: Arc<AppProvingKey<VC>>,
+        app_exe: Arc<NonRootCommittedExe>,
+        agg_stark_pk: AggStarkProvingKey,
```

Comment:
we should update `agg_keygen`: there should be one version that only generates `AggStarkProvingKey` and a separate one that also does `halo2`. we can discuss naming offline

Reply:
similarly we should polish the CLI flow for e2e stark without halo2